index.html

```diff
@@ -1,14 +1,15 @@
 <!DOCTYPE html xmlns="http://www.w3.org/1999/xhtml" xmlns:fb="http://ogp.me/ns/fb#">
     <head>
         <meta http-equiv="content-type" content="text/html; charset=UTF8" />
-        <meta name="description" content="Electricity Map | Real-time visualisation of the climate footprint of electricity generation" />
+        <meta name="description" content="Electricity Map | Clean vs. dirty electricity production in Europe" />
```

Comment:
My thoughts regarding the original were:

a) Main message that comes true is "This is real-time", while IMO it should be "different countries have different climate footprints related to energy production". Being real-time is a neat detail, definitely helps with PR and is a good show-off but for the average User it doesn't matter that much. In other words, it's important that we mention it but I don't think it has to be on the headline. 

 b) Felt a little too long. I think titles should be shorter whenever possible but, in this case, the long title also had some visual impact. When I added it on the map it was taking too much space.

Other suggestions could be:
- `Climate impact of electricity production in Europe`
- `CO2 footprint of electricity production in Europe`

In any case, I don't think this is something worth discussing further. Here's my opinion but I trust you to make a good judgment.

Let me know what I show change in the PR.

Cheers,
Gui